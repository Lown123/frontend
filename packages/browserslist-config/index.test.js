const browserslist = require('browserslist')
const { name: NPM_PACKAGE } = require('./package.json')
const config = require('./')

describe(NPM_PACKAGE, () => {
  it('should export an array', () => {
    expect(Array.isArray(config)).toBe(true)
  })

  it('should not throw an error when extended', () => {
    // eslint-disable-next-line global-require
    jest.doMock(NPM_PACKAGE, () => require('./'), { virtual: true })

    expect(() => browserslist([`extends ${NPM_PACKAGE}`])).not.toThrow()
  })
})
