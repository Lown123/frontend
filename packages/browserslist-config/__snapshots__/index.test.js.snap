// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@zonedigital/browserslist-config should not contain invalid queries 1`] = `
Array [
  "and_chr 70",
  "chrome 70",
  "edge 18",
  "firefox 63",
  "ie 11",
  "ie_mob 11",
  "ios_saf 12",
  "ios_saf 11.3-11.4",
  "ios_saf 11.0-11.2",
  "ios_saf 10.3",
  "ios_saf 10.0-10.2",
  "safari 12",
  "safari 11.1",
  "safari 11",
  "safari 10.1",
  "safari 10",
]
`;
